compiler/damlc/daml-doc/src/DA/Daml/Doc/Extract/Templates.hs

```diff
@@ -161,18 +162,30 @@ isIfaceChoice decl@ClsInstDecl{}
 -- present, else nothing.
 hasImplementsConstraint :: ClsInstDecl GhcPs -> Maybe (RdrName, HsType GhcPs)
 hasImplementsConstraint (XClsInstDecl _) = Nothing
-hasImplementsConstraint ClsInstDecl{..}
-  | (L _ [L _ ctx], L _ ty) <- splitLHsQualTy $ hsSigType cid_poly_ty
```

Comment:
I noticed that the doc tests were failing since the docstrings for interface choices weren't being generated. The solution was to change this function to expect more than one constraint in its context

Reply:
Nice, thank you!